### PR TITLE
Create es_es.json

### PR DESCRIPTION
**Pull request by albertosaurio65**

### CODE REVIEW

File: src/main/resources/assets/mermod/lang/es_es.json [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+  "item.mermod.moon_rock_modifier": "Roca lunar",
+  "item.mermod.ursula_shell_modifier": "Concha de Úrsula",
+  "tooltip.mermod.modifierHint": "Mantenga presionada la tecla Mayús y haga clic con el botón derecho para eliminar los modificadores",
+  "tooltip.mermod.canBeDyed": "Se puede teñir",
+  "config.mermod.name": "Configuración Mermod",
+  "config.mermod.option.swimSpeed": "Efecto de velocidad de natación",
+  "config.mermod.option.swimSpeedMultiplier": "Multiplicador de velocidad de natación",
+  "config.mermod.option.waterBreathing": "Efecto de respiración de agua",
+  "config.mermod.option.nightVision": "Efecto de visión nocturna",
+  "config.mermod.option.nightVisionFlashingFix": "Corrección de parpadeo de visión nocturna"
```
Comment (Kichura) [inline src/main/resources/assets/mermod/lang/es_es.json]: Line 16 (nightVisionFlashingFix) will get overridden by #1 if that one was merged - just to be clear about it.

**Comment by albertosaurio65** [comment]
delete it?

**Comment by Kichura** [comment]
> delete it?

no need to - just needs to be updated to reflect #1's english file.

**Comment by albertosaurio65** [comment]
i say to delete de line 16

**Pull request merged**